helix-contract/contracts/ln/base/LnBridgeTarget.sol

```diff
@@ -0,0 +1,130 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import "../interface/ILnBridgeSource.sol";
+import "./LnBridgeHelper.sol";
+
+contract LnBridgeTarget is LnBridgeHelper {
+    uint256 constant public MIN_REFUND_TIMESTAMP = 30 * 60;
+    struct TransferInfo {
+        uint48 nonce;
+        uint48 lastRefundNonce;
```

Comment:
Maybe it is simpler to use `lastRefundTransferId` instead of `lastRefundNonce` with the compromise of loosing gas opt.

Reply:
TransferInfo => FillInfo